leap_year.rb

```diff
@@ -0,0 +1,18 @@
+    leap_years = []
+    print "What year would you like to start with?"
+    start_year = gets.chomp.to_i
+   
+    print "What year would you like to end with?"
+    end_year = gets.chomp.to_i
+    
+    for year in start_year..end_year
+        if year % 400 == 0 || (year % 4 == 0 && (year % 100 != 0))
+            leap_years.push(year)
+        else
+            next
+        end
```

Comment:
You don't need the `else` `next` at all because there's nothing after the `if` block.

```suggestion
        if year % 400 == 0 || (year % 4 == 0 && (year % 100 != 0))
            leap_years.push(year)
        end
```